code/mob/dead/observer.dm

```diff
@@ -411,6 +431,23 @@
 	if (!O)
 		return null
 
+	var/is_mutantrace = FALSE
+	if (istype(src.mutantrace, /datum/mutantrace/lizard))
+		O.icon_state = "ghost-lizard"
+		is_mutantrace = TRUE
+	else if (istype(src.mutantrace, /datum/mutantrace/cow))
+		O.icon_state = "ghost-cow"
+		is_mutantrace = TRUE
+	else if (istype(src.mutantrace, /datum/mutantrace/skeleton))
+		O.icon_state = "ghost-skeleton"
+		is_mutantrace = TRUE
+	else if (istype(src.mutantrace, /datum/mutantrace/roach))
+		O.icon_state = "ghost-roach"
+		is_mutantrace = TRUE
+	else if (istype(src.mutantrace, /datum/mutantrace/pug))
+		O.icon_state = "ghost-pug"
+		is_mutantrace = TRUE
```

Comment:
ditto

Reply:
Could be, would be the only location it's used tho, considering the above comment